docs/Schema.sql

```diff
@@ -0,0 +1,60 @@
+ /*
+    The tables are in the order they will need to be created in our code dude to references.
+ */
+
+
+ /*
+    This is table on each university in the United States. study_set and user_info will 
+    link to it because it is better than having all the information about a university 
+    in multiple tables
+*/
+CREATE TABLE university(
+    id INT PRIMARY KEY,
```

Comment:
I would update all these to `id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY` so that the ID is generated automatically. Originally I said to use `SERIAL` but looks like that's an outdated feature.